test/run_test.py

```diff
@@ -1018,7 +1018,9 @@ def get_selected_tests(options):
         )
 
     if options.functorch:
-        selected_tests = FUNCTORCH_TESTS
+        selected_tests = list(
+            filter(lambda test_name: test_name in FUNCTORCH_TESTS, selected_tests)
+        )
```

Comment:
Use list comprehensions
```suggestion
        selected_tests = [tname for tname in selected_test if tname in FUNCTORCH_TESTS]
```